operator/pkg/gateway-api/secret.go

```diff
@@ -64,9 +64,9 @@ func (r *secretSyncer) notInSecretsNamespace() builder.Predicates {
 	}))
 }
 
-func (r *secretSyncer) enqueueTLSSecrets() handler.EventHandler {
+func enqueueTLSSecrets(c client.Client) handler.EventHandler {
 	return handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, obj client.Object) []reconcile.Request {
-		scopedLog := r.logger.WithFields(logrus.Fields{
+		scopedLog := log.WithContext(ctx).WithFields(logrus.Fields{
```

Comment:
The global logger was already used before the PR. It was just this commit that reverted it again. So not really a regression. But i agree that it would be better to pass the logger as parameter in this case.

-> Changed